src/main/rst/02_selenium_ide.rst

```diff
@@ -894,7 +894,7 @@ the ``xpath=`` label when specifying an XPath locator.
 These examples cover some basics, but in order to learn more, the 
 following references are recommended:
 
-* `W3Schools XPath Tutorial <http://www.w3schools.com/Xpath/>`_ 
+* `W3Schools XPath Tutorial <http://www.w3schools.com/xsl/xpath_intro.asp>`_ 
```

Comment:
I would rather remove the reference to w3schools, let's find a different site for a tutorial